### PR TITLE
Split logics by classes

### PR DESCRIPTION
**Pull request by ihorsas**
Add:
1. ThreadManager
2. ElevetorManager
3. Visualizer thread
4. Constants class

Update:
1.  move thread classes to separate package
2. toString methods
3. logic of stoping elevator and waiting till request
4. infinitive loop (soon we'll create a separate thread with UI to communicate with threads in real-time)

### CODE REVIEW

File: src/main/java/com/lnu/src/manager/ElevatorManager.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import com.lnu.src.strategy.ElevatorStrategy;
+import com.lnu.src.strategy.StopsOnEveryFloorStrategy;
+import com.lnu.src.thread.Elevator;
+
+import java.util.List;
+
+public class ElevatorManager {
+
+  private final ElevatorStrategy stopsOnDemandStrategy = new StopsOnEveryFloorStrategy();
+  private final ElevatorStrategy stopsOnEveryFloorStrategy = new StopsOnEveryFloorStrategy();
```
Comment (tsmichael) [inline src/main/java/com/lnu/src/manager/ElevatorManager.java]: I am not okey with it. I think we do not need there " = new Some_Strategy()"

File: src/main/java/com/lnu/src/manager/ElevatorManager.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    this.floors = floors;
+  }
+
+  public Elevator createStopsOnDemandElevator() {
+    return new Elevator(floors, stopsOnDemandStrategy);
+  }
+
+  public Elevator createStopsOnEveryFloorElevator() {
+    return new Elevator(floors, stopsOnEveryFloorStrategy);
+  }
```
Comment (tsmichael) [inline src/main/java/com/lnu/src/manager/ElevatorManager.java]: init new strategy inside those methods

File: src/main/java/com/lnu/src/thread/PassengerRandomizer.java [hunk 3]
```diff
@@ -9,15 +9,15 @@
...
+public class PassengerRandomizer extends Thread {
 
   private static final Logger LOGGER = LogManager.getLogger();
   private final float chance;
   private final int time;
   private final List<Floor> floors;
 
 
-  PassengerGenerator(float chance, int time, List<Floor> floors) {
+  public PassengerRandomizer(float chance, int time, List<Floor> floors) {
```
Comment (tsmichael) [inline src/main/java/com/lnu/src/thread/PassengerRandomizer.java:20]: Why Randomizer not Generator?

Comment (ihorsas) [reply src/main/java/com/lnu/src/thread/PassengerRandomizer.java:20]: cause we've got such name in our UML diagram

**Review by tsmichael — APPROVED** [review]
Look good. Thanks for it!

**Pull request merged**